### PR TITLE
Regenerate video_intelligence and provide a synth script

### PR DESCRIPTION
**Pull request by dazuma**
Add a synth script for generating video_intelligence, and regenerate the library.

There are some minor changes, mostly to documentation. The only significant change seems to be the moving of the credentials class into the version namespaces. For now I left the old credentials class in place, with a "deprecated" comment on it. Question for people: is it safe to remove the class? I'm not sure if it is considered part of the public surface.

### CODE REVIEW

**Comment by jbolinger** [comment]
I believe the strategy for the un-versioned credentials file is to keep it around for GA gems but otherwise drop it.

**Comment by blowmage** [comment]
> Question for people: is it safe to remove the class? I'm not sure if it is considered part of the public surface.

It is part of the public API. For gems that have gone 1.0 we will maintain the file path and constant. If you want I can push a commit to this PR to do that.

**Comment by dazuma** [comment]
Okay, sgtm. The current commit does maintain the file path and constant (and just adds a `@deprecated` comment). So I think this should be reviewable as-is then.

File: google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta1.rb [hunk 1]
```diff
@@ -13,13 +13,15 @@
...
+require "google/cloud/videointelligence/v1beta1/video_intelligence_pb"
+require "google/rpc/status_pb"
 
 module Google
   module Cloud
     # rubocop:disable LineLength
 
     ##
-    # # Ruby Client for Google Cloud Video Intelligence API ([GA](https://github.com/GoogleCloudPlatform/google-cloud-ruby#versioning))
+    # # Ruby Client for Google Cloud Video Intelligence API ([Alpha](https://github.com/GoogleCloudPlatform/google-cloud-ruby#versioning))
```
Comment (blowmage) [inline google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta1.rb]: Video Intelligence is `GA`. Why is it being downgraded to `Alpha`?

Comment (dazuma) [reply google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta1.rb]: @jbolinger Do you know why this is? It's happening for the v1beta1 and v1beta2 clients but not the v1 client.

Comment (jbolinger) [reply google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta1.rb]: @dazuma it's controlled by a setting (Alpha is the default value). See https://github.com/googleapis/googleapis/blob/ce7d34ee23cf759b9c323d902a067ea4bd1537f4/google/cloud/videointelligence/v1/videointelligence_gapic.yaml#L14

Comment (dazuma) [reply google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta1.rb]: @jbolinger That file appears to be generated as well. Do you know what the source of truth is?

Comment (jbolinger) [reply google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta1.rb]: @dazuma yes. Sorry, I assumed you were familiar with that part of the process. I will ping you directly with the details.

Comment (dazuma) [reply google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta1.rb]: Thanks @jbolinger for the info. @blowmage I will get this fixed upstream and will update this PR. Slowly figuring out the process... :)

Comment (dazuma) [reply google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta1.rb]: Committed a fix.

**Pull request merged**